build.gradle

```diff
@@ -107,6 +119,29 @@ List<String> resolveIntelliJVersions(String versionKey) {
   return [versionValue]
 }
 
+String checkForNewEap (String gradleMajor){
```

Comment:
I wouldn't call the param `gradleMajor` as it falsely suggests it's about the version of Gradle itself 🤔
Maybe `intellijMajorVersion`?